apis/v1/controllers/addresses.controller.js

```diff
@@ -1,10 +1,12 @@
 // ---------------
 // Based Imports
 // ---------------
-const Address = require('../../schemas/address.schema');
-const { JSONResponse } = require('../../utilities/response.utility');
+const Address = require('../../../schemas/address.schema');
+const { JSONResponse } = require('../../../utilities/response.utility');
 
-const User = require('../../schemas/user.schema');
+const User = require('../../../schemas/user.schema');
+
+const mongoose = require('mongoose');
 // ---------------
 
```

Comment:
There should be a function here to get all addresses by user_id. 
This function should use the user id passed to the route to search addresses for all addresses with this user_id attached.